src/java.desktop/share/classes/sun/java2d/cmm/ProfileDataVerifier.java

```diff
@@ -106,7 +106,7 @@ private static int readInt32(byte[] data, int off) {
      */
     private static final int MAX_TAG_COUNT = 100;
 
-    private static final int HEADER_SIZE = 128;
+    public static final int HEADER_SIZE = 128;
     private static final int TOC_OFFSET = HEADER_SIZE + 4;
     private static final int TOC_RECORD_SIZE = 12;
```

Comment:
Minor: All the constants are defined at the end of file, is it better to move them to the beginning ?

Reply:
Perhaps someday when we go for this kind of unification